src/ccutil/unicharset.cpp

```diff
@@ -815,41 +819,64 @@ bool UNICHARSET::load_via_fgets(
     float advance = 0.0f;
     float advance_sd = 0.0f;
     // TODO(eger): check that this default it ok
-    // after enabling BiDi iterator for Arabic+Cube.
+    // after enabling BiDi iterator for Arabic.
```

Comment:
Maybe it should be Arabic+LSTM ?

Reply:
Maybe. The more important point is that this TODO is either still open, or the comment should be removed.